### ISSUE TITLE
inconsistency between CWL input and janis input for File type

### ISSUE THREAD

**Issue by beccyl** [PMCC-BioinformaticsCore/janis-assistant#50]
I've written a tool in janis which takes a file input (inp).  I have generated the CWL, and can run this successfully with cwl-runner / or RabixComposer/Executor.  There appears to be inconsistency in what janis assistant accepts as input, for File type, versus cwl-runner for File type.  It would be nice to have these consistent for when working in different environments.  (I am unfamiliar with WDL equivalence).

The underlying purpose is to run CWL in the Seven Bridges cloud environment.

valid json input for File type in CWL
```
{
    "inp": {
        "class": "File",
        "path": "/path/to/file.bam"
    }
}
```

However when run with janis assistant, it expects "inp" to be type string:
```
{
   "inp": "/path/to/file.bam"
}
```

Janis python code:
```
            ToolInput(
                tag="inp",
                input_type=Bam(),
                prefix="--INPUT",
                separate_value_from_prefix=True,
                doc=InputDocumentation(
                    doc="(-I) Input file SAM/BAM to extract reads from. Required."
                ),
            ),
```
The exception generated by "janis run" is:
Exception: Couldn't qualify the filename for the input 'inp' ({'class': 'File', 'path': '/path/to/file.bam'}) to type '<janis_bioinformatics.data_types.bam.Bam object at 0x7fd86d8ef0f0>': expected string or bytes-like object

**Comment by illusional** [comment]
Hey @beccyl, this is a pretty reasonable idea to try and be generic enough to accept both. 

I've added a pre-processor (https://github.com/PMCC-BioinformaticsCore/janis-assistant/commit/606a16f58a76752604ba184d9ecbfe9e37d4f41c) that will unwrap CWL input Files / Directories to get it into a shape that Janis supports. This is released in `v0.11.3`, and it's feasible to add a WDL one too, but unless their's demand I'll hold off.

**Comment by illusional** [comment]
Closing due to v0.11.3 release, feel free to reopen if it's still an issue.
